### ISSUE TITLE
Accessibility:Panel: Name of the button “Panel Title” is not read when accessed in backward navigation using Shift+Tab key.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3494]
Description: Name of the button “Panel Title” is not read when accessed in backward navigation using Shift+Tab key.

Expected: Name of the UI should be read in backward navigation also. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94650979-4d3d6d80-0315-11eb-9497-687b15d593e1.png)
